setup.cfg

```diff
@@ -71,18 +71,12 @@ zun.image.driver =
 
 zun.network.driver =
     kuryr = zun.network.kuryr_network:KuryrNetwork
-    cni = zun.network.cni_network:ZunCNI
```

Comment:
I think these should be merged a bit better probably. The k8s driver can be listed after the cni one. Same comment for dropping the cni.binding section. It will make future rebases harder.